src/main/asciidoc/json-pointers.adoc

```diff
@@ -0,0 +1,22 @@
+== Json Pointers
+
+Vert.x provides an implementation of <a href="https://tools.ietf.org/html/rfc6901">Json Pointers from RFC6901</a>.
+You can use pointers both for querying and for writing. You can build your {@link io.vertx.core.json.pointer.JsonPointer} using
+a string, a URI or manually appending paths:
+
+[source,java]
+----
+{@link examples.JsonPointerExamples#example1Pointers}
+----
+
+After instantiating your pointer, you can use method {@link io.vertx.core.json.pointer.JsonPointer#queryJson(java.lang.Object)} to query
+a JsonObject/JsonArray. If you want to write inside a JsonObject/JsonArray use {@link io.vertx.core.json.pointer.JsonPointer#writeJson(java.lang.Object, java.lang.Object)}:
```

Comment:
JSON object or array : in JS JsonObject does not make sense

Reply:
_You can update a JSON object or array thanks to_

Reply:
maybe is better saying a JSON value, since this function manages JSON objects, arrays but also primitives

Reply:
If you have a root pointer and you pass a primitive, let's say a string, to the query function it returns the string (it's a valid operation for json pointers)